cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/BuilderUtils.java

```diff
@@ -85,11 +85,15 @@ public static void compileWithTestsAndWrite(Path sourceRootPath, boolean offline
         List<BLangPackage> packages = compiler.build();
 
         if (skiptests) {
-            outStream.println();
-            compiler.write(packages);
+            if (packages.size() > 0) {
+                outStream.println();
+                compiler.write(packages);
+            }
         } else {
-            Utils.testWithBuild(sourceRootPath, null);
-            compiler.write(packages);
+            if (packages.size() > 0) {
+                Utils.testWithBuild(sourceRootPath, null);
+                compiler.write(packages);
+            }
```

Comment:
Shouldnt we give a message saying that theres nothing to build ?

Reply:
Added a message with 0ee1aad